lib/ansible/plugins/callback/counter_enabled.py

```diff
@@ -152,7 +152,7 @@ def v2_runner_on_ok(self, result):
 
         delegated_vars = result._result.get('_ansible_delegated_vars', None)
 
-        if self._play.strategy == 'free' and self._last_task_banner != result._task._uuid:
+        if getattr(self._play.strategy, 'FLOW', 'lockdown')  == 'free' and self._last_task_banner != result._task._uuid:
```

Comment:
As of right now, I believe that `self._play.strategy` is just a string name of the strategy.  It's not an object.